CONTRIBUTING.md

```diff
@@ -157,6 +172,12 @@ For commands that create or modify complex data structures, the use of configura
 
 Note: we are experimenting with a generator, make sure to try it out in [tools/cli-generator](./tools/cli-generator/)
 
+### API Interactions
+
+Atlas CLI and MongoDB CLI use [go-client-mongodb-atlas](https://github.com/mongodb/go-client-mongodb-atlas/) 
```

Comment:
this may be stale now, go-sdk?

Reply:
Good point! will extend it to cover both SDKs.